docs: add screenshot requirement to PR template (#199)
open-sauced/guestbook: .github/PULL_REQUEST_TEMPLATE.md

- [ ] 📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot

> <!-- Please provide a screenshot of your profile being generated on the README. This will make it easier for maintainers when reviewing PRs and ensure that you to ran the generate command as mentioned in the previous question. -->

Reviewer: ```suggestion
<!-- Please provide a screenshot of your profile being generated on the README. This ensures that you run the `npm run contributors:generate` command, as mentioned in the previous question, making it easier for maintainers to review PRs. -->
```
